pkg/api/handlers/libpod/manifests.go

```diff
@@ -90,6 +90,16 @@ func ManifestInspect(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	if len(schema2List.Manifests) == 0 {
```

Comment:
@restanrm Since no new imports are being added would it be possible to drop vendor changes. It should work with current mods as well right ?

Reply:
Ok, no problem

Reply:
I was trying to fix the CI.

Reply:
@restanrm it might be failing due to other reasons.